packages/astro/src/core/pipeline.ts

```diff
@@ -75,58 +32,19 @@ export class Pipeline {
 	 *
 	 * It throws an error if the page can't be rendered.
 	 */
-	async #tryRenderRoute(
-		renderContext: Readonly<RenderContext>,
-		env: Readonly<Environment>,
-		mod: Readonly<ComponentInstance> | undefined,
-		onRequest?: MiddlewareHandler
+	async renderRoute(
```

Comment:
I suppose the docs are outdated now, because it seems that this function doesn't throw any error anymore? We should update it

Reply:
Do you know when that behavior changed?